### PR TITLE
APEXCORE-506 Added check for trailing whitespaces

### PR DESCRIPTION
**Pull request by vrozov**
@tweise, @chandnisingh Please review

### CODE REVIEW

File: codestyle-config/pom.xml [hunk 1]
```diff
@@ -22,16 +22,18 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0"
     xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
     xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
   <parent>
-    <groupId>org.apache.apex</groupId>
```
Comment (tweise) [inline codestyle-config/pom.xml:26]: Please remove the parent pom changes. They are not related to this change and can be taken up separately (while also changing Apex parent pom).

**Comment by tweise** [comment]
The checkstyle rule seems to work fine. I changed all violations:

https://github.com/tweise/apex-core/commit/1ef25421970826de4e2f4615ed04934f5ffb55ad

Will raise PR once this is pulled in. Please rebase.

File: codestyle-config/pom.xml (hunk 1 above)
Comment (vrozov) [reply codestyle-config/pom.xml:26]: I can move them into a separate commit, but we can't combine changes to the code style and the parent pom into a single PR.

Comment (tweise) [reply codestyle-config/pom.xml:26]: I don't see why two different issues should be in a single PR. I would suggest to wrap up the checkstyle change and then address parent pom properly in a separate JIRA.

**Comment by vrozov** [comment]
rebased to the latest master

**Pull request merged**